webapp-admin/src/components/SystemUserForm.js

```diff
@@ -188,6 +202,20 @@ class SystemUserForm extends Component {
                             })}
                             </select>
                         </label>
+                        <label>Limit this system-user to a set of serial numbers:&nbsp;
+                            <button className="p-button--brand" title="Add serial number" onClick={this.handleAddEmptySerialField}>
```

Comment:
I would suggest using [p-button--neutral](https://vanillaframework.io/docs/patterns/buttons#neutral) as the orange button here is to strong.

The button would also benifit from being more [dense](https://vanillaframework.io/docs/patterns/buttons#dense) and [inline](https://vanillaframework.io/docs/patterns/buttons#inline).

```html
<button className="p-button--neutral is-dense is-inline" title="Add serial number" onClick={this.handleAddEmptySerialField}>
```

Reply:
That looks much better now!